src/components/ChatApp/SearchField.react.js

```diff
@@ -1,19 +1,44 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { TextField, IconButton as _IconButton } from '@material-ui/core';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import SearchIcon from '@material-ui/icons/Search';
-import UpIcon from '@material-ui/icons/ArrowUpward';
-import DownIcon from '@material-ui/icons/ArrowDownward';
+import _CloseIcon from '@material-ui/icons/Close';
+import _UpIcon from '@material-ui/icons/ExpandLess';
+import _DownIcon from '@material-ui/icons/ExpandMore';
 
 const ESCAPE_KEY = 27;
 const F_KEY = 70;
 
+const NavigationIconStyle = css`
+  width: 0.8rem;
+  height: 0.8rem;
```

Comment:
Change the width and height to 1rem. Currently it looks too small.
Rest LGTM 🚀

Reply:
Let's wait for @akshatnitd review as well